packages/generator/src/imports.ts

```diff
@@ -41,11 +42,13 @@ export function externalImportDeclaration(
 }
 
 export function coreImportDeclaration(
-  namedImports: string[]
+  namedImports: string[],
+  oDataVersion: 'v2' | 'v4'
 ): ImportDeclarationStructure {
+  const moduleSuffix = oDataVersion === 'v4' ? '/v4' : '';
   return {
     kind: StructureKind.ImportDeclaration,
-    moduleSpecifier: '@sap-cloud-sdk/core',
+    moduleSpecifier: `@sap-cloud-sdk/core${moduleSuffix}`,
```

Comment:
If I search for `@sap-cloud-sdk/core/v4` I do not find a thing. I would expect that the generated test eintities need to import the request builders etc. from the v4 path?

Reply:
Or is there some test for this?

Reply:
This is because this currently does not generate a v4 version for the type tests. I will add this to the todo list. You can find a v4 in the generated test-utils of core, though